Northstar.CustomServers/mod/scripts/vscripts/evac/_evac.gnut

```diff
@@ -363,6 +365,7 @@ void function Evac( int evacTeam, float initialWait, float arrivalTime, float wa
 	EmitSoundOnEntity( dropship, evacShip.flyoutSound )
 	
 	// holster all weapons
+        file.shouldholsterweapons[ dropship ] <- true
```

Comment:
whitespace issue (spaces i assume)

Reply:
on the line im commenting on, the one that was added

Reply:
<img width="415" height="67" alt="image" src="https://github.com/user-attachments/assets/2dfde500-e82a-4d4c-9565-f3ad64b67cd6" />

lol

Reply:
Oh, you meant tabs vs spaces